src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import Tab from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected(tab: Tab): void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const foundSelectedTab = tabs.some(tab => selectedTabId === tab.id);
+
+  return (
+    <div>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, i) => (
+            <li
+              className={selectedTabId === tab.id || (!foundSelectedTab && !i)
+                ? 'is-active'
+                : ''}
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                onClick={event => {
+                  event.preventDefault();
+
+                  if (tab.id !== selectedTabId) {
+                    onTabSelected(tab);
+                  }
+                }}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="tab-content">
+        {tabs.find(tab => selectedTabId === tab.id)?.content}
```

Comment:
```suggestion
        {tabs.find(tab => selectedTabId === tab.id)?.content || tabs[0].content}
```
<img width="829" alt="image" src="https://user-images.githubusercontent.com/55802630/185154706-f674e458-93d1-4162-ac44-0185aea80913.png">